gridcapa-cse-app/src/main/java/com/farao_community/farao/gridcapa_cse/app/configurations/AmqpConfiguration.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.gridcapa_cse.app.configurations;
+
+import com.farao_community.farao.gridcapa_cse.app.CseListener;
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.listener.MessageListenerContainer;
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * @author Amira Kahya {@literal <amira.kahya at rte-france.com>}
+ */
+@Configuration
+public class AmqpConfiguration {
```

Comment:
Queue should not be an anonymous queue, so it is not defined by a queue-prefix but just a queue name. As we would have replicated services they must be plugged on same queue to be sure the message is not consumed twice.
So property "cse-request.queue" is enough.

CseRequest exchange is a fanout exchange and has no routing key. 

You can see gridcapa-task-manager I think it is more how it should look like for AMQP config.